xbmc/video/VideoInfoScanner.cpp

```diff
@@ -1847,6 +1874,8 @@ namespace VIDEO
           for (guide = candidates->begin(); guide != candidates->end(); ++guide)
           {
             auto title = guide->cScraperUrl.GetTitle();
+            if (title.empty())
+              title = guide->strTitle;
```

Comment:
add a space after this, or wrap the condition with brackets without add the space